### DISCUSSION TITLE
General Discussion

### DISCUSSION THREAD

**Discussion by CodeAsherPlayz** [PythonMathCalculator/python-math-calculator#40] [General]
This discussion can be used to talk about anything in the world 😉

**Comment by CodeAsherPlayz** [comment]
@rutvikpadhiyar000 please reply to this thread when you have time after finishing college 😄

**Reply by ghost** [reply]
hey sorry i war inactive for so long i just got busy in collage work

**Reply by CodeAsherPlayz** [reply]
It's okay ;D
